internal/exec/util/passwd.go

```diff
@@ -99,6 +99,8 @@ func (u Util) EnsureUser(c types.PasswdUser) error {
 			strconv.FormatUint(uint64(*c.UID), 10))
 	}
 
+	args = appendIfTrue(args, c.NonUnique, "--non-unique")
```

Comment:
If a config specifies two users with the same UID, and one of them specifies `nonUnique: true` and one of them doesn't, what should happen?  The answer can't depend on the order of the entries, since Ignition configs are declarative.